packages/react/src/components/CTA/CTA.js

```diff
@@ -48,7 +41,7 @@ const CTA = ({ style, type, customClassName, ...otherProps }) => {
    */
   const getVideoData = useCallback(async () => {
```

Comment:
Is there a way to externalize this hook?

Reply:
yep done